book/src/installation-binaries.md

```diff
@@ -29,15 +29,19 @@ a portable `x86_64` binary.
 
 1. Go to the [Releases](https://github.com/sigp/lighthouse/releases) page and
    select the latest release.
-1. Download the `lighthouse-${VERSION}-x86_64-unknown-linux-gnu-portable.tar.gz` binary.
-1. Extract the archive:
-    1. `cd Downloads`
-    1. `tar -xvf lighthouse-${VERSION}-x86_64-unknown-linux-gnu.tar.gz`
+1. Download the `lighthouse-${VERSION}-x86_64-unknown-linux-gnu.tar.gz` binary. For example, to obtain the binary file for v4.0.1 (the latest version at the time of writing), a user can run the following commands in a linux terminal:
+    ```bash
+    cd ~
+    curl -LO https://github.com/sigp/lighthouse/releases/download/v4.0.1/lighthouse-v4.0.1-x86_64-unknown-linux-gnu.tar.gz
+    tar -xvf lighthouse-v4.0.1-x86_64-unknown-linux-gnu.tar.gz
+    ```
 1. Test the binary with `./lighthouse --version` (it should print the version).
-1. (Optional) Move the `lighthouse` binary to a location in your `PATH`, so the `lighthouse` command can be called from anywhere.
-    - E.g., `cp lighthouse /usr/bin`
-
-> Windows users will need to execute the commands in Step 3 from PowerShell.
+1. (Optional) Move the `lighthouse` binary to a location in your `PATH`, so the `lighthouse` command can be called from anywhere. For example,
+```bash
+cp lighthouse /usr/bin
```

Comment:
```suggestion
sudo cp lighthouse /usr/bin
```

This should probably be `sudo`. Very few systems would have user-writable `/usr/bin` directories